### DISCUSSION TITLE
Moving to systray

### DISCUSSION THREAD

**Discussion by oleghcp** [icsharpcode/ILSpy#2329] [Ideas]
Could you implement option that will allow moving app to systray instead of window closing?

**Comment by christophwille** [comment]
Technically, yes, but why? ILSpy isn't (in most cases, anyways) used like a TSR (think chat apps), but task-oriented. What sort of difference would that make especially that for most people that would mean opening the tray & hunting and pecking for a small ILSpy icon.

**Reply by oleghcp** [reply]
I use ILSpy as extension for Visual Studio and open it from context menu of ide to show certain code. But ILSpy is opened a bit long fist time. In this case ILSpy works faster if already opened. But I often close ILSpy window accidentally. It would be very convenient if it moved to systray instead of exiting.

**Reply by oleghcp** [reply]
No?

░░░░▄▄██▀▀▀▀▀▀▀█▄░░░░░░░
░░▄▀▀░░░░░░░░░░░░▀█▄░░░░
░█▀░░░░░░░░░░░░░░░░█▄░░░
▄█░░░▄▄▄█▄▄░░▄░░░░░░█░░░
█░░░▀██▀▀▀░██▄█░░░░░▀█░░
█░░░░░▄░░▀▄▄░░░░▄░░░░▀█░
█▀░████░░░░████░▀░░░░░█░
█░░▀███░░░░███▀░░░░░░░█▄
█░░▄░▀▀░░░░▀▀░░▄░░░░░░██
▀█░░█▀░▄▄░░█░▀▀░░░░░░░█▀
░▀█░░░░█▀▀▀▀▀▀▄░░░░░░░█░
░░█░░░█░░▄▄▄▄░▄█░░░░░▄█░
░░▀█░░██▀▀██▀▀▀█▀░░░▄█░░
░░░▀█▄░░░░░░░░▄░░░░▄█░░░
░░░░░▀▄▄▀██████░░▄█▀░░░░
░░░░░░░▀█▄▄▄▄▄▄▄█▀░░░░░░
░░░░░░░░░▄▄████▄░░░░░░░░
